From 8eb91f5d6f8e954568ef98e9796fe0116f302dc7 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Ignacy=20Kuchci=C5=84ski?= <ignacykuchcinski@gmail.com>
Date: Thu, 19 Jun 2025 11:48:55 +0200
Subject: [PATCH 4/5] cups-base: Add root system group

Currently we only include wheel in system groups, which prevents root
requests from being authenticated.

Fix that by adding root to the system groups.
---
 elements/components/cups-base.bst | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/elements/components/cups-base.bst b/elements/components/cups-base.bst
index 93efebdfd..80d43601a 100644
--- a/elements/components/cups-base.bst
+++ b/elements/components/cups-base.bst
@@ -17,7 +17,7 @@ variables:
     --with-tls=openssl
     --with-cups-group=lp
     --with-pkgconfpath="%{libdir}/pkgconfig"
-    --with-system-groups=wheel
+    --with-system-groups="root wheel"
     --with-rundir=/run/cups
     localedir=/usr/share/locale
     DSOFLAGS="$CFLAGS $LDFLAGS"
-- 
2.49.0

